packages/flex-dev-utils/src/fs.ts

```diff
@@ -182,6 +196,43 @@ export const checkFilesExist = (...files: string[]): boolean => {
   return files.map(fs.existsSync).every((resp) => resp);
 };
 
+/**
+ * Calculates the sha of a file
+ * @param paths
+ */
+/* istanbul ignore next */
+export const calculateSha256 = async (...paths: string[]): Promise<string> => {
+  return new Promise((resolve, reject) => {
+    const shasum = crypto.createHash('sha256');
+    const stream = createReadStream(path.join(...paths));
+
+    stream.on('data', (data) => shasum.update(data));
+    stream.on('error', (err) => reject(err));
+    stream.on('end', () => resolve(shasum.digest('hex')));
+  });
+};
+
+/**
+ * Removes a file
+ * @param paths
+ */
+export const removeFile = (...paths: string[]): void => unlinkSync(path.join(...paths));
+
+/**
+ * Copies from from src to dest
+ * @param srcPaths
+ * @param destPaths
+ */
+export const copyFile = (srcPaths: string[], destPaths: string[]): void =>
+  copyFileSync(path.join(...srcPaths), path.join(...destPaths));
+
+/**
+ * Checks the provided file exists
+ *
+ * @param paths the paths to the file
+ */
+export const checkAFileExist = (...paths: string[]): boolean => existsSync(path.join(...paths));
```

Comment:
`checkAFileExists` or just `fileExists`

Reply:
actually think `fileExists` is better 😛

Reply:
We already have `filesExist` which has a different implementation - I purposely named it this otherwise it will be so confusing to have `filesExist` and `fileExists` :p

Reply:
ok finee then `checkFileExists`